### PR TITLE
Onboarding and offboarding checklists

### PR DESCRIPTION
**Pull request by adborden**
Add checklists as GH issue templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/onboard-team-member.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - [ ] #datagov-notifications
+  - [ ] #opp-data-analytics
+- [ ] Add new team member to email lists
+  - [ ] [Data.gov support list](https://groups.google.com/a/gsa.gov/forum/#!forum/datagov)
+  - [ ] [Data.gov team list](https://groups.google.com/a/gsa.gov/forum/#!forum/datagovhelp)
+- [ ] Make sure team member has 2FA enabled for their GitHub account and [request](https://github.com/GSA/GitHub-Administration/blob/master/README.md#requesting-access-to-the-gsa-organization) membership to GSA GitHub org
+- [ ] Add team member to GitHub teams
+  - [ ] [data-gov-admins](https://github.com/orgs/GSA/teams/data-gov-admin/members) (PMO only)
+  - [ ] [data-gov-support](https://github.com/orgs/GSA/teams/data-gov-support/members) (non-PMO)
+- [ ] Request TTS Bug Bounty access [#bug-bounty-partners](https://gsa-tts.slack.com/messages/C5JQCD9PH)
```
Comment (pburkholder) [inline .github/ISSUE_TEMPLATE/onboard-team-member.md:35]: Add to TTS bug bounty to  offboarding?

File: .github/ISSUE_TEMPLATE/offboard-team-member.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- [ ] Remove access from [GSA GitHub org](https://github.com/GSA/GitHub-Administration/blob/master/README.md#removing-access-to-the-gsa-organization) (if leaving GSA)
+- [ ] Remove team member from [AWS sandbox access](https://github.com/GSA/datagov-infrastructure-live/tree/master/iam#new-users)
+- [ ] Remove team member from [jumpbox operators](https://github.com/GSA/datagov-deploy/blob/develop/ansible/group_vars/all/vault.yml)
+- [ ] Remove team member from Data.gov email lists
+  - [ ] [Data.gov support list](https://groups.google.com/a/gsa.gov/forum/#!forum/datagov)
+  - [ ] [Data.gov team list](https://groups.google.com/a/gsa.gov/forum/#!forum/datagovhelp)
+- [ ] Remove team member from Uptrends via [email](https://docs.google.com/spreadsheets/d/1Z9Zpr1mpx-65i_fH2VTbVofPtidpLZs5cnkO0Jz53Vc/edit#gid=0)
+- [ ] Remove team member from [New Relic](https://newrelic.com)
+- [ ] Remove team member from [Docker Hub](https://cloud.docker.com/orgs/datagov/teams)
+- [ ] Remove team member from [Data.gov admin accesses](https://github.com/GSA/datagov-account-management) following procedure
```
Comment (pburkholder) [inline .github/ISSUE_TEMPLATE/offboard-team-member.md:23]: - [ ] Rotate Ansible Vault ?

**Review by pburkholder — APPROVED** [review]
Nice! I added a couple of suggestions. Good to merge when you're good with it.

**Comment by adborden** [comment]
Thanks, added your suggestions.

**Pull request merged**